extensions/azurePublishNew/src/api.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
We'll get rid of this in favor of Natalie's service layer so you can delete it from this PR.